readme.md

```diff
@@ -59,30 +54,28 @@ Getting the code (optionally, you can create a workspace):
     export PATH=$PATH:$GOPATH/bin
     go get -t github.com/zalando/skipper
 
-Build:
+#### Building
+[needs more context copy]
 
     cd src/github.com/zalando/skipper
     go install ./cmd/skipper
 
-Test:
+#### Testing
 
     go test ./...
 
-### Contributing
-We welcome contributions to this project.
-
-### License
-
-Copyright 2015 Zalando SE
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
+### Additional Documentation
+Skipper's [godoc](https://godoc.org/github.com/zalando/skipper) page includes detailed information on these topics:
+- The Routing Mechanism
+- Matching Requests
+- Filters - Augmenting Requests
+- Service Backends
+- Route Definitions
+- Data Sources
+- Extending It with Customized Predicates, Filters, and Builds
+- Proxy Packages
+- Logging and Metrics
+- Performance Considerations
+
+### Contributing/TODO
+We welcome contributions to this project. [Need to add TODO's, any guidelines/formatting preferences.]
```

Comment:
I agree. We'll discuss this with the current maintainers, and make a pull request with a proposal soon. Is it ok if this happens in a next round?